packages/app/vite.config.js

```diff
@@ -6,7 +6,9 @@ import { defineConfig } from 'vite';
 const dirname = fileURLToPath(new URL('.', import.meta.url));
 
 const [pkg, libPkg, sharedPkg] = ['.', '../lib', '../shared'].map((prefix) =>
-  JSON.parse(fs.readFileSync(path.resolve(dirname, `${prefix}/package.json`)))
+  JSON.parse(
+    fs.readFileSync(path.resolve(dirname, `${prefix}/package.json`), 'utf-8')
```

Comment:
The only linting / type-checking errors reported were on this line in a few of the `vite.config.js` files. TS wants `JSON.parse` to receive a string, but eslint-plugin-unicorn says it works fine with a buffer... I decided to go with TS' recommendation, since we disable the ESLint rule is a more targetted solution than `@ts-ignore` or `as string`.